tests/.gitignore

```diff
@@ -1 +1 @@
-output
+output
```

Comment:
not sure if we need this change it's the same content

Reply:
I do not even understand why is this change here. In any case, we have now one .gitignore and that is in `tests/` folder. For now it will exclude `node_modules` from `playwright` and from what I got locally also from future's argos structure. Otherwise is ignores `output` folder recursively in actual directory and in any subfolder, same for `dist`.  We could add more specific rules later in shared or divided them in particular subfolder, right?

Reply:
but `node_modules` and `dist` are excluded at the root level
https://github.com/containers/podman-desktop/blob/main/.gitignore#L1

https://github.com/containers/podman-desktop/blob/main/.gitignore#L3

so not sure why you need to exclude them again ?

Reply:
ah, perfect then.